e2e/support/helpers/e2e-dragndrop-helpers.js

```diff
@@ -0,0 +1,39 @@
+// Rely on native drag events, rather than on the coordinates
+// We have 3 "drag-handles" in this test. Their indexes are 0-based.
```

Comment:
we were using this helper function in the pivot tables unit test, and it works perfectly here, so I extracted it to use multiple places.

I think this function is based on [this github issue.](https://github.com/cypress-io/cypress/issues/3942#issuecomment-485648100)